Pod/Classes/BONTag.h

```diff
@@ -0,0 +1,69 @@
+//
+//  BONTag.h
+//  Pods
+//
+//  Created by Nora Trapp on 5/11/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "BONCompatibility.h"
+
+@protocol BONTextable;
+
+@interface BONTag : NSObject <NSCopying>
+
+@property (copy, nonatomic, readonly, BONNonnull) NSString *startTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *endTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *escapeString;
+@property (copy, nonatomic, readonly, BONNonnull) id<BONTextable> textable;
+
+- (BONNonnull instancetype)initWithStartTag:(BONNonnull NSString *)startTag endTag:(BONNonnull NSString *)endTag escapeString:(BONNonnull NSString *)escapeString textable:(BONNonnull id<BONTextable>)textable;
+- (BONNonnull instancetype)initWithTag:(BONNonnull NSString *)tag textable:(BONNonnull id<BONTextable>)textable;
+
+@end
+
+/**
+ *  Create a tag with a custom start/end/escape combination.
+ *
+ *  @param startTag     The start tag.
+ *  @param endTag       The end tag.
+ *  @param escapeString The escape string.
+ *  @param textable     The style to apply.
+ *
+ *  @return A @p BONTag instance representing the tag.
+ */
+CG_INLINE BONTag *BONCNonnull BONTagComplexMake(NSString *BONCNonnull startTag, NSString *BONCNonnull endTag, NSString *BONCNonnull escapeString, id<BONTextable> BONCNonnull textable)
```

Comment:
Good question without any answer really. Can change to `NS_INLINE` if you prefer, what they do is identical.